service/src/main/kotlin/fi/espoo/evaka/emailclient/EmailClient.kt

```diff
@@ -55,4 +55,14 @@ class EmailClient(private val client: AmazonSimpleEmailService) : IEmailClient {
             logger.warn("Will not send email due to invalid toAddress: (traceId: $traceId)")
         }
     }
+
+    private fun checkWhitelist(address: String): Boolean {
+        val isWhitelisted = whitelist == null || whitelist.any { it.matches(address) }
```

Comment:
Q: wouldn't `whitelist?.any { it.matches(address) } ?: true` do the same and be more idiomatic than explicit null checking?

Reply:
Yeah, I guess, I just wanted to be very explicit that if there is not whitelist it's ok.

Reply:
Yeah, totally up to you. More "wondering" than "questioning" on my part.

Reply:
But I guess it's pretty obvious anyway.